_assets/js/netlify/topics/generateChildElements.js

```diff
@@ -0,0 +1,30 @@
+const notAllowed = ['em', 'strong', 'u', 'i', 'b', 'ul', 'blockquote', 'table', 'thead', 'td', 'tr','tfooter', 'tbody', 'br', 'input'];
+
+const generateChildElements = (html) =>
+  html.map((item) => {
+    if (item.children && item.children.length > 0) {
+      if (notAllowed.includes(item.nodeName.toLowerCase())) {
+        return h(
+          item.nodeName.toLowerCase(),
+          { "id": item.id.toString(), "className": item.className },
+          generateChildElements([ ...Array.from(item.children)])
+        )
+      } else {
+        return h(
+          item.nodeName.toLowerCase(),
+          { "id": item.id.toString(), "className": item.className },
+          item.innerHTML,
+          generateChildElements([ ...Array.from(item.children)])
+        )
+      }
+  } else if (!notAllowed.includes(item.nodeName.toLowerCase())) {
+    return h(
+      item.nodeName.toLowerCase(),
+      { "id": item.id.toString(), "className": item.className },
+      item.innerHTML
+    )
+  }
```

Comment:
This code is a bit confusing because there's a few things going on at once:
- Dealing with name, id, and className
- Dealing with item content (if allowed)
- Dealing with the item's children

wdyt about separating those into three steps, and then calling `h` on the result?

```suggestion
    const isItemAllowed = !notAllowed.includes(item.nodeName.toLowerCase());
    const itemHasChildren = item.children?.length > 0;

    const content = [];
    if (isItemAllowed) content.push(item.innerHTML);
    if (itemHasChildren) {
      content.push(generateChildElements([ ...Array.from(item.children)]));
    }
    // This should never happen with a filter, but to be safe!
    if (!content.length) return;
    
    return h(
          item.nodeName.toLowerCase(),
          { "id": item.id.toString(), "className": item.className },
          ...content,
    );
```